ops/staging-deploy.tmpl.yaml

```diff
@@ -145,12 +167,14 @@ extraEnvVars: &envVars
     value: /app/fits/tools/mediainfo/linux
   - name: NEGATIVE_CAPTCHA_SECRET
     value: $NEGATIVE_CAPTCHA_SECRET
+  - name: OMP_THREAD_LIMIT
+    value: "1"
   - name: PASSENGER_APP_ENV
     value: production
   - name: RAILS_CACHE_STORE_URL
     value: redis://:$REDIS_PASSWORD@adventist-knapsack-staging-redis-master:6379/staging
   - name: RAILS_ENV
-    value: production
+    value: staging
```

Comment:
there is no config/environments/staging.rb in Hyku right now. Maybe there should be, but there isn't. its safe to use production here.